content/ytt/docs/latest/file-marks.md

```diff
@@ -65,15 +65,19 @@ Change type of file, affecting how `ytt` processes it.
 --file-mark '<path>:type=<file-type>'
 ```
 
-By default `file-type` is determined based on file extension. 
+By default `file-type` is determined based on file extension (and, as of v0.32.0, content).
```

Comment:
```suggestion
By default `file-type` is determined based on file extension and, as of v0.32.0, content.
```

I think my preference is to drop the parens. It seems like important information, especially as we release more versions in the future.